cime_config/tests.py

```diff
@@ -101,6 +101,7 @@
             "ERS_Ld5.ne4_oQU240.F2010.eam-rrtmgpxx",
             "REP_Ln5.ne4_oQU240.F2010",
             "SMS_Ld9.ne4pg2_oQU480.F2010.eam-thetahy_sl_pg2_mass",
+            "ERP_Ld9.ne4_ne4.FIDEAL",
```

Comment:
Due to heavier output on day 6, this test is running into a MEMLEAK fail on Crusher+gnu (and pm-cpu + gnu). Is it okay to add a test-mod to `xmlchange PIO_ROOT=1`: i.e. change test to `ERP_Ld9.ne4_ne4.FIDEAL.allactive-pioroot1`?
```
 tStamp_write: model date =   00010104       0 wall clock = 2022-08-13 01:22:15 avg dt =     0.35 dt =     0.34
 memory_write: model date =   00010104       0 memory =     754.54 MB (highwater)        121.70 MB (usage)  (pe=    0 comps= cpl ATM LND ICE OCN GLC ROF WAV IAC ESP)
 tStamp_write: model date =   00010105       0 wall clock = 2022-08-13 01:22:15 avg dt =     0.32 dt =     0.24
 memory_write: model date =   00010105       0 memory =     754.54 MB (highwater)        122.21 MB (usage)  (pe=    0 comps= cpl ATM LND ICE OCN GLC ROF WAV IAC ESP)
 Write restart file at   00010106       0
(seq_rest_write)  write rpointer file rpointer.drv
 (seq_io_wopen)  create file ERP_Ld9.ne4_ne4.FIDEAL.crusher_gnu.JNextIntegration20220813_010620.cpl.r.0001-01-06-00000.nc
 Restart filename: ERP_Ld9.ne4_ne4.FIDEAL.crusher_gnu.JNextIntegration20220813_010620.cpl.r.0001-01-06-00000.nc
 tStamp_write: model date =   00010106       0 wall clock = 2022-08-13 01:22:16 avg dt =     0.32 dt =     0.32
 memory_write: model date =   00010106       0 memory =    1172.77 MB (highwater)        157.24 MB (usage)  (pe=    0 comps= cpl ATM LND ICE OCN GLC ROF WAV IAC ESP)
 tStamp_write: model date =   00010107       0 wall clock = 2022-08-13 01:22:16 avg dt =     0.30 dt =     0.21
 memory_write: model date =   00010107       0 memory =    1178.77 MB (highwater)        157.30 MB (usage)  (pe=    0 comps= cpl ATM LND ICE OCN GLC ROF WAV IAC ESP)
```

Reply:
https://github.com/E3SM-Project/E3SM/issues/5076